pkg/controller/infinispan/resources/service/service_controller.go

```diff
@@ -99,37 +99,54 @@ func Add(mgr manager.Manager) error {
 
 func (s serviceResource) Process() (reconcile.Result, error) {
 	if s.infinispan.HasSites() {
-		if err := s.reconcileResource(computeSiteService(s.infinispan)); err != nil {
+		siteService := computeSiteService(s.infinispan)
+		s.infinispan.AddLabelsForServices(siteService.Labels)
+		s.infinispan.AddOperatorLabelsForServices(siteService.Labels)
+		if err := s.reconcileResource(siteService); err != nil {
 			return reconcile.Result{}, err
 		}
 	}
 
 	service := computeService(s.infinispan)
+	s.infinispan.AddLabelsForServices(service.Labels)
+	s.infinispan.AddOperatorLabelsForServices(service.Labels)
 	setupServiceForEncryption(s.infinispan, service)
 	if err := s.reconcileResource(service); err != nil {
 		return reconcile.Result{}, err
 	}
 
-	if err := s.reconcileResource(computePingService(s.infinispan)); err != nil {
+	pingService := computePingService(s.infinispan)
+	s.infinispan.AddLabelsForServices(pingService.Labels)
```

Comment:
It would be nice to use `(s serviceResource) computeXYXService` and place labeling inside it

Reply:
@dmvolod you mean create a method for serviceResource and put all the code there?

Reply:
No, @rigazilla it would be nice to put `s.infinispan.AddLabelsForServices(pingService.Labels)` and other labeling methods inside `computeXYXService` methods as it's related to constricting Service structure

Reply:
yep, probably labelling will always be part of the object construction